pkgs/by-name/go/gotree/package.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  pname = "gotree";
+  version = "0.1.0";
```

Comment:
Mind updating to version [0.2.0](https://github.com/elbachir-one/gt/releases/tag/v0.2.0), which was released recently.

Please also update the commit message, the PR title, and `src.hash` below 🙂